### DISCUSSION TITLE
Cortex Analyzers missing?

### DISCUSSION THREAD

**Discussion by lonewaffle** [Security-Onion-Solutions/securityonion#6858] [Unsupported Versions] [answered]
Hello!

Using SO 2.3.90 here. I'm just now getting around to checking out Cortex and there are no analyzers showing up for any organization.
![Screen_Shot_2022-01-13_at_8_52_47_AM](https://user-images.githubusercontent.com/5062130/149365036-b5084dfe-aa46-4dea-966f-894db8e4bd2d.png)

It's not that they aren't enabled. It seems that they're not in there at all. I've checked in the container and all of the analyzers and responders are in /Cortex-Analyzers just as they should be but they don't seem to be getting picked up for some reason.

Someone mentioned on another forum about needing to open up the permissions on these directories to get them to work, but that's part of the container, so I'm thinking any changes to that won't persist across restarts. 

Any thoughts on how I can get them to show up?

UPDATE:
I haven't made any changes at all to the conf files for Hive or Cortex, but decided to take a look at cortex-application.conf to see if anything looked out of place. Here's the relevant sections:

--------------------------------
```
## ANALYZERS
#
analyzer {
  # Absolute path where you have pulled the Cortex-Analyzers repository.
  path = ["/Cortex-Analyzers/analyzers"]

  # Sane defaults. Do not change unless you know what you are doing.
  fork-join-executor {

    # Min number of threads available for analysis.
    parallelism-min = 2

    # Parallelism (threads) ... ceil(available processors * factor).
    parallelism-factor = 2.0

    # Max number of threads available for analysis.
    parallelism-max = 4
  }
}

## RESPONDERS
##
responder {
  # Directory that holds responders
  urls = ["/Cortex-Analyzers/responders", "/custom-responders"]

  fork-join-executor {
    # Min number of threads available for analyze
    parallelism-min = 2
    # Parallelism (threads) ... ceil(available processors * factor)
    parallelism-factor = 2.0
    # Max number of threads available for analyze
    parallelism-max = 4
  }
}

# It's the end my friend. Happy hunting!
```
----------------------------

I'm noticing that the analyzers use "path" and the responders use "urls". Is it supposed to be like that? Does this look different than anyone else's?

**Comment by dougburks** [comment, marked as answer]
From https://docs.securityonion.net/en/2.3/cortex.html#usage:
> Users initially authenticate to Cortex via the username and password supplied during setup. Once authenticated to the Cortex organization, users will possess superadmin privileges, capabable of managing all organizations, users, etc.
>
> From here, users should create an additional user for the SecurityOnion organization, or create their own organization/users if they wish to log into Cortex and manage analyzers and responders.
>
> It is always recommended that you create your own organization, but the provided organizations should work for testing.

Also note that Cortex and TheHive will be going away soon:
https://blog.securityonion.net/2022/01/coming-soon-in-security-onion-23100.html

**Reply by lonewaffle** [reply]
Thanks for letting me know about the coming changes. I won't spend any more time on this issue since it will be removed soon anyway.
